jobs/azure-discovery/src/main/java/com/tmobile/pacbot/azure/inventory/collector/WebAppInventoryCollector.java

```diff
@@ -4,6 +4,7 @@
 import com.microsoft.azure.management.Azure;
 import com.microsoft.azure.management.appservice.DefaultErrorResponseException;
 import com.microsoft.azure.management.appservice.WebApp;
+import com.tmobile.pacbot.azure.inventory.ErrorManageUtil;
```

Comment:
Reformat file please using ide format file

Reply:
Addressed in commit